backend/src/main/kotlin/trie/RunCreateTrie.kt

```diff
@@ -0,0 +1,197 @@
+package trie
+
+import persistence.PlaceRepository
+import java.io.BufferedOutputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStream
+import java.util.stream.Collectors
+import java.util.zip.GZIPOutputStream
+import kotlin.system.measureTimeMillis
+
+val OUTPUT_PATH = ".\\backend\\src\\main\\resources\\US.trie"
```

Comment:
Kann des sein dass der Pfad nu von ana alten location is? hat bei mir ned funktioniert (ah ne dnachdem i die backslashes zu slashes umgschriebn hab) -> src/main/resources/US.trie wars :)